entries/c/consentmanager.net.json

```
@@ -0,0 +1,12 @@
+{
+  "Consent Manager": {
+    "domain": "consentmanager.net",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.consentmanager.net/books/cmp/page/working-with-two-factor-authentication",
+    "categories": [
+      "other"
```

```suggestion
      "legal"
```

Like it was noted in #8283, the legal category might be a better fit.